roles/atomic_scan_verify/tasks/main.yml

```diff
@@ -9,18 +9,36 @@
 #   - atomic install
 #   - atomic scan
 #
+- name: Set scanner image (RHEL)
+  set_fact:
+    scanner_image: "registry.access.redhat.com/rhel7/openscap"
+    scanner_type: "openscap"
+  when: ansible_distribution == "RedHat"
+
+- name: Set scanner image
+  set_fact:
+    scanner_image: "docker.io/fedora/atomic_scan_openscap"
+    scanner_type: "atomic_scan_openscap"
+  when: ansible_distribution != "RedHat"
+
+- name: Set scanner target
+  set_fact:
+    scanner_target: "registry.access.redhat.com/rhel7"
 
 - name: Install openscap
-  command: atomic install registry.access.redhat.com/rhel7/openscap
+  command: atomic install {{ scanner_image }}
   register: result
 
 - name: Check installation of openscap
   fail:
     msg="Atomic install was unsuccessful"
   when: "'Installation complete' not in result.stdout"
 
+- name: Pull scanner target
+  command: atomic pull {{ scanner_target }}
```

Comment:
Can we pull a rhel7 image from somewhere else?  I know that is not ideal but little else we can do for now.